### ISSUE TITLE
ts3server doesn't start with updateonstart

### ISSUE THREAD

**Issue by xopez** [GameServerManagers/LinuxGSM#2897]
## User Story

ts3server doesn't start with updateonstart turned on.

## Basic info

* **Distro:** Ubuntu 18.04
* **Game:** TeamSpeak 3
* **Command:** update, start
* **LinuxGSM version:** v20.3.0

## Further Information

It just checks the version currectly, but doesn't end the start command as in earlier versions.
Output with updateonstart turned on:
```
ts3server@ts:~$ ./ts3server r
[  OK  ] Stopping ts3server: Graceful: CTRL+c: 2: OK
[  OK  ] Updating ts3server: Checking for update: teamspeak.com

No update available
* Local build: 3.12.1
* Remote build: 3.12.1


ts3server@ts:~$
```
without updateonstart:
```
ts3server@ts:~$ ./ts3server r
[ INFO ] Stopping ts3server: TeamSpeak 3 Server is already stopped
[  OK  ] Starting ts3server: TeamSpeak 3 Server
ts3server@ts:~$
```

## To Reproduce

Steps to reproduce the behaviour:
1. Turn updateonstart to on
2. start ts3

## Expected behaviour
Start no matter what updateonstart is.

**Comment by dgibbs64** [comment]
Thanks this will be a priority

**Comment by h3o66** [comment]
Fixed with the release 20.3.1
